Avoid using vendored OpenSSL.

Dependency chain:

xwin -> native-tls -> openssl-sys -> openssl-src

--- cargo-crates/xwin-0.2.14/Cargo.toml.orig	2023-08-06 19:00:27 UTC
+++ cargo-crates/xwin-0.2.14/Cargo.toml
@@ -133,6 +133,5 @@ native-tls = [
 default = ["rustls-tls"]
 native-tls = [
     "ureq/native-tls",
-    "native-tls-crate/vendored",
 ]
 rustls-tls = ["ureq/tls"]
